src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 13px;
+  height: 12px;
+  margin: 2px 5px 2px 2px;
+}
+
+.stars__star-active {
+  background-image: url(images/star-active.svg);
+  width: 13px;
+  height: 12px;
+  margin: 2px 5px 2px 2px;
```

Comment:
after you use nth-child, these styles can be deleted from here, cause it's default styles for each star, you need to change only background-image for active stars

```suggestion
```